### PR TITLE
Fix #355 and #420

### LINKED ISSUE #355: Complete Alumni page
Right now it's just a map and then a bunch of quotes. Shouldn't we have some sort of introduction, even if brief, and a general summary of how 1418 prepares members for the future?

### PR DESCRIPTION
**Pull request by Git-HorizonZz**
-written an intro for the alumni page

### CODE REVIEW

File: css/alumni.sass [hunk 1]
```diff
@@ -5,15 +5,27 @@
...
     max-width: 150px
     max-height: auto
     display: block
     margin-left: auto
     margin-right: auto
- 
-    
+
+.li-quotes
+    list-style: none
```
Comment (ErikBoesen) [inline css/alumni.sass:17]: I would advocate that instead we simply remove the ul/li tags and target `blockquote` elements directly. At the very least, this class should be called `quote` or something, as you can just say `li.quote` to specify li elements—no need for including the tag name in the class

File: css/alumni.sass [hunk 2]
```diff
@@ -5,15 +5,27 @@
...
+    list-style-type: none
+    padding: 0em
+    margin: 0em
 
 main p
     font-size: 20px
     margin-left: 30px
+
+#intro-p
```
Comment (ErikBoesen) [inline css/alumni.sass:26]: Same thing here—no need to include the tag in the ID name

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: css/alumni.sass (hunk 1 above)
Comment (Git-HorizonZz) [reply css/alumni.sass:17]: done 👍

File: css/alumni.sass [hunk 3]
```diff
@@ -5,15 +5,31 @@
...
     max-height: auto
     display: block
     margin-left: auto
     margin-right: auto
- 
-    
+
+li.quotes
+    list-style: none
+    list-style-type: none
```
Comment (ErikBoesen) [inline css/alumni.sass:18]: Why even use a list if we're removing all the style? How about just plain blockquotes?

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: css/alumni.sass (hunk 3 above)
Comment (Git-HorizonZz) [reply css/alumni.sass:18]: I tried using just blockquotes, but that removes indentation

Comment (ErikBoesen) [reply css/alumni.sass:18]:
```sass
blockquote
    margin-left: 40px
```

Comment (ErikBoesen) [reply css/alumni.sass:18]:
```sass
blockquote
    margin-left: 40px
```

Comment (Git-HorizonZz) [reply css/alumni.sass]: I by switched up my words there, but that works. The <li>s are gone

**Review by ErikBoesen — APPROVED** [review]

**Pull request merged**